internal/pkg/manager.go

```diff
@@ -34,8 +57,8 @@ func (m *ManagerConfig) Run() error {
 	}
 
 	// use desired config on start
-	if err := m.updateConfigToLatest(); err != nil {
-		m.Logger.Error("failed to update the config", "error", err)
+	if err := m.updateConfigToLatest(viper.GetString("haproxy.config.base")); err != nil {
```

Comment:
we shouldn't use viper outside of the `cmd` package - this should get added to the `ManagerConfig` maybe?